zulip.md

```diff
@@ -0,0 +1,68 @@
+# Zulip
+
+##  Getting started on Zulip:
+
+### If you’re new here, here’s what you can do to get started with this community:
+
+1.  :heavy_check_mark: Introduce yourself on [#newcomers](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) stream in the [#newcomers > Introductions](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers/topic/Introductions) topic and tell us your motivations for joining so we can get to know you and help you on your journey.
```

Comment:
Suggestion: `share with us your motivations for joining so we can get to know and help you on the open source journey`